gradle/libs.versions.toml

```diff
@@ -8,7 +8,8 @@ assertj = "3.22.0"
 atomikos = "5.0.8"
 awaitility = "4.2.0"
 aws = "2.17.288"
-bouncyCastle = "1.70"
+bouncycastle-jdk18on = "1.72"
+bouncyCastle = "1.70" # this bouncycastle version is deprecated in favor of the jdk18on
```

Comment:
is the typo ("C" vs "c") intentional? I'm asking, because if it is not, and you intended to write `bouncycastle` and `bouncycastle-jdk18on` it won't work, because the "-" will act as a separator, thus making `bouncycastle` a leaf and a node at the same time, which is not allowed.
If it was indeed intentional, I would suggest some other means of distinguishment, like `"bouncycastlejdk18on` and switching names sometime in the future.